samza-azure/src/main/java/org/apache/samza/ProcessorEntity.java

```diff
@@ -38,11 +39,13 @@ public ProcessorEntity(String jobModelVersion, String processorId) {
     this.partitionKey = jobModelVersion;
     this.rowKey = processorId;
     this.isLeader = false;
+    Random rand = new Random();
+    this.liveness = rand.nextInt(10000) + 2;
```

Comment:
Maybe I misunderstood the liveness value's meaning here. What is the random value used for? The length of the heartbeat interval in seconds? Or just a random value? If not for the length of heartbeat interval, what's the significance of the random value here?